test/note-route-test.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
```

Comment:
This is a great start, there are a few more cases you'll need to test for but the structure of these additional tests will be similar to those you've implemented already :)